src/components/para_trials/counter.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import styles from "../../styles/Paraflow.module.css";
+
+interface CounterProps {
+  title: string;
+  maxCount: number;
+  minCount: number;
+  color: string;
```

Comment:
this should probably be a const union, so something like
```suggestion
  color: "yellow" | "green" | "blue";
```

to help whoever's calling it realize the correct values (types can be a great way to help document code)

Reply:
I wasn't sure how we would be implementing color choices, so I left it open for future changes. But this is a great tip that I'll add in for now. Thanks!